webserver.cc

```diff
@@ -49,7 +49,8 @@ Server::Server(configArguments configArgs, std::map<std::string, std::vector<std
     signals.async_wait(boost::bind(&Server::stop, this));
 
     ip::tcp::resolver resolver(io_service);
-    ip::tcp::resolver::query query(IPADDR, std::to_string(configArgs.port));
+    ip::tcp::resolver::query query("0.0.0.0", std::to_string(configArgs.port));
```

Comment:
it might be worthwhile to put the ip address in the config file, so that we can change it easily